src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -62,6 +62,19 @@ std::shared_ptr<IPassManager> BasePass::getPassManager() {
     return sharedMgr;
 }
 
+/**
+ * @brief Perform addition of two blobs values
+ */
+static void SumBlobs(Blob::Ptr& src_blob, Blob::Ptr& dst_blob) {
+    IE_ASSERT(src_blob != nullptr);
+    IE_ASSERT(dst_blob != nullptr);
+    IE_ASSERT(src_blob->size() == dst_blob->size());
+
+    float* src_blob_buf = src_blob->buffer().as<float*>();
```

Comment:
is this always f32 type behind the buffer?

Reply:
Good point.-
Added code, which supports multiple types, depending on precision specified in TensorDesc.